rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
What is the purpose of this file's inclusion? If this is some sort of temporary thing, provide some kind of context for why it's here and what the plan is.